src/app/beer_garden/router.py

```diff
@@ -443,35 +455,69 @@ def _determine_target_garden(operation: Operation) -> str:
     raise Exception(f"Bad operation type {operation.operation_type}")
 
 
-def _forward_http(operation: Operation, conn_info: dict):
+def _forward_http(operation: Operation, target_garden: Garden):
     """Actually forward an operation using HTTP
 
     Args:
         operation: The operation to forward
         conn_info: Connection info
     """
+
+    conn_info = target_garden.connection_params
     endpoint = "{}://{}:{}{}api/v1/forward".format(
         "https" if conn_info.get("ssl") else "http",
         conn_info.get("host"),
         conn_info.get("port"),
         conn_info.get("url_prefix", "/"),
     )
 
-    if conn_info.get("ssl"):
-        http_config = config.get("entry.http")
-        return requests.post(
-            endpoint,
-            data=SchemaParser.serialize_operation(operation),
-            cert=http_config.ssl.ca_cert,
-            verify=http_config.ssl.ca_path,
-        )
+    response = None
 
-    else:
-        return requests.post(
-            endpoint,
-            data=SchemaParser.serialize_operation(operation),
-            headers={"Content-type": "application/json", "Accept": "text/plain"},
+    try:
+        if conn_info.get("ssl"):
+            http_config = config.get("entry.http")
+            response = requests.post(
+                endpoint,
+                data=SchemaParser.serialize_operation(operation),
+                cert=http_config.ssl.ca_cert,
+                verify=http_config.ssl.ca_path,
+            )
+
+        else:
+            response = requests.post(
+                endpoint,
+                data=SchemaParser.serialize_operation(operation),
+                headers={"Content-type": "application/json", "Accept": "text/plain"},
+            )
+
+        if response.status_code != 200:
+            publish(
+                Event(
+                    name=Events.GARDEN_UNREACHABLE.name,
+                    payload_type=operation.schema,
+                    payload=operation,
+                    error_message=f"Attempted to forward operation to garden "
+                    f"'{operation.target_garden_name}' but the connection returned an error code of "
+                    f"{response.status_code}. Please talk to your system administrator.",
+                )
+            )
+        elif target_garden.status != "RUNNING":
+            beer_garden.garden.update_garden_status(target_garden.name, "RUNNING")
+
+        return response
+
+    except Exception as ex:
+        publish(
+            Event(
+                name=Events.GARDEN_ERROR.name,
+                payload_type=operation.schema,
+                payload=operation,
+                error_message=f"Attempted to forward operation to garden "
+                f"'{operation.target_garden_name}' but an error occurred."
+                f"Please talk to your system administrator.",
+            )
         )
+        raise ex
```

Comment:
If you're re-raising an exception like this the statement needs to be just `raise`, not `raise ex`. The way it is now is going to be reported in the stacktraces as the exception handling for the first exception encountering a problem and raising a *new* exception (that happens to be identical to the first one). That's how we get these long chained exceptions with "error encountered while processing exception..." in the logs.